csml_engine/src/lib.rs

```diff
@@ -88,6 +91,88 @@ pub fn get_open_conversation(client: &Client) -> Result<Option<DbConversation>,
     get_latest_open(client, &mut db)
 }
 
+/**
+ * create bot state
+ * 
+ * { "statusCode": i32, "body": version_id }
```

Comment:
this function should only return the version_id string

then the server (lambda, csml_server...) should format the return body as an object: {version_id: XXXX}

Reply:
(FYI success code for create bot version should be 201, not 200: https://httpstatuses.com/201)